### DISCUSSION TITLE
Joy of Victory

### DISCUSSION THREAD

**Discussion by Sol-au-fein** [secretcowlevel/DoomtrooperPublic#82] [Card Balance]
Joy of Victory (GENERAL)
POSTCOMBAT
If your combatant survived combat: 5D are added to your D pool. If the warrior was WOUNDED in the combat, it is healed.

➡️ If your combatant survived and wounded or killed enemy warrior in combat

This card is called joy of victory. What victory is that, when your warrior is injured and lost combat, without touching the opponent. Free healing and free DP, just wait until your opponent attacks and do nothing. Doesnt make sense.

**Comment by TomekPL-1** [comment]
I agree with Sol-au-fein, but I would suggest to reduce reward to 3 DP

Joy of Victory (GENERAL)
POSTCOMBAT
If your combatant survived and wounded or killed enemy warrior in combat: 3D are added to your D pool. If the warrior was WOUNDED in the combat, it is healed.

**Reply by Sol-au-fein** [reply]
Yeah! That was also my suggestion, but nobody agreed.
Sure, i prefer 3DP version.

**Reply by 21-ric** [reply]
I prefer 3 DP too.
